deploy/ray-operator/api/v1alpha1/raycluster_types.go

```diff
@@ -86,21 +86,30 @@ type Extension struct {
 	// Pod docker image
 	Image string `json:"image,omitempty"`
 
+	//create operation should have no idList (generate by controller)
+	IdList []string `json:"idList"`
+
 	// Logical groupName for worker in same group, it's used for heterogeneous feature to distinguish different groups.
 	// GroupName is the unique identifier for the pods with the same configuration in one Ray Cluster
 	GroupName string `json:"groupName,omitempty"`
 
 	// Command to start ray
 	Command string `json:"command,omitempty"`
 
+	// Args to start ray
+	Args string `json:"args,omitempty"`
+
 	// Labels for pod, raycluster.component and rayclusters.ray.io/component-name are default labels, do not overwrite them.
 	// Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users,
 	// but do not directly imply semantics to the core system. Labels can be used to organize and to select subsets of objects.
 	Labels map[string]string `json:"labels,omitempty"`
 
-	// The service acccount name.
+	// The service account name.
 	ServiceAccountName string `json:"serviceAccountName,omitempty"`
 
+	// The container ports
+	Ports []v1.ContainerPort `json:"ports,omitempty" patchStrategy:"merge" patchMergeKey:"containerPort" protobuf:"bytes,6,rep,name=ports"`
```

Comment:
In out future plan, we can also support this configuration in template, right?

Reply:
Yes, we will allow users to set in the template.